packages/paste-website/src/pages/roadmap/index.mdx

```diff
@@ -32,57 +32,7 @@ export const pageQuery = graphql`
 
 <Box>
 
-## Release 0.4
-
-Release date: December 3, 2019
-
-**Design releases**
-
-- Flex tokens will be added to our Design System Sketch files.
-- Alerts will be design-ready and built in an upcoming release. 🔔
-- Grids will be design-ready and built in an upcoming release.
-
-**Engineering releases**
-
-- Buttons will be productionized 🎉 and usable on SendGrid / Console.
-- Anchors will be productionized too! 🔗
-- The icon system will also be on the production train 🚝
-
-**Other**
-
-- The theme switcher will work as intended, allowing all styles to be viewable based on SendGrid and Console Themes.
-
-
----
-
-## Release 0.5
-
-Release date: January 14, 2020
-
-**Design releases**
-
-- Design and ship the first set of form field components (Label, Input, Textarea, and Inline error). 📝
-- Design more form field components (Select, Checkbox and Checkbox group, Radio and Radio Group, and Toggle). 📝
-- Design a Button group component.
-- Apply unified design language for alpha.
-
-**Engineering releases**
-
-- Ship an Alert component. 🔔
-- Ship a Grid component.
-- Ship more Typography components (Heading and Lists).
-- Implement continuous deployment of Paste packages. 
-
-**Other**
-
-- Add an icon list to the [icon system guidelines](/icon-system) with more design guidelines. Right now, you can access our  
-  <Anchor href="https://twilio-labs.github.io/paste/?path=/story/overview-icons--list">
-    full icon list on Storybook
-  </Anchor>.
-
----
-
-## Release 0.6
+## Release 2020.02.25
 
 Release date: February 25, 2020
 
```

Comment:
```
**Design releases**

- Design more form field components (Select, Checkbox and Checkbox group, and Radio and Radio Group). 📝
- Design a Form composition component. 📝
- Design a Toast component. 🍞

**Engineering releases**

TBD

**Other**

- Add an icon list to the [icon system guidelines](/icon-system) with more design guidelines. Right now, you can access our  
  <Anchor href="https://twilio-labs.github.io/paste/?path=/story/overview-icons--list">
    full icon list on Storybook
  </Anchor>.
- Make this website responsive 📱
```

Reply:
> - Make this website responsive 📱
Yeah, right 😛

Reply:
idk drop a grid on it??

ok fine, you're right. Take it out.

Reply:
removed